manual_tests/tuning_contrast_limits_from_api.py

```diff
@@ -0,0 +1,332 @@
+import argparse
+import json
+from pathlib import Path
+
+import numpy as np
+from cryoet_data_portal import Client, Tomogram
+from neuroglancer import viewer_state
+from neuroglancer.url_state import to_url
+from scipy.spatial.transform import Rotation
+from screenshot_urls import run_screenshot_loop
+
+from cryoet_data_portal_neuroglancer.io import load_omezarr_data
+from cryoet_data_portal_neuroglancer.precompute.contrast_limits import (
+    CDFContrastLimitCalculator,
+    ContrastLimitCalculator,
+    GMMContrastLimitCalculator,
+    combined_contrast_limit_plot,
+    compute_contrast_limits,
+)
+from cryoet_data_portal_neuroglancer.state_generator import combine_json_layers, generate_image_layer
+
+# Set up logging - level is info
+# logging.basicConfig(level=logging.INFO, force=True)
+
+OUTPUT_FOLDER = "/media/starfish/LargeSSD/data/cryoET/data/FromAPI"
```

Comment:
This output folder is really specific to a file system, a default value for the output folder should perhaps be more generic :D (like targeting the current working directory or something like this)